unstructured/ingest/main.py

```diff
@@ -338,7 +344,13 @@ def main(
     verbose,
     metadata_include,
     metadata_exclude,
+    fields_include,
 ):
+    if "metadata" not in fields_include and (metadata_include or metadata_exclude):
+        logger.error(
```

Comment:
```suggestion
        logger.warning(
```

nit: this is really more of a warning